jupyterhub/templates/pod-culler/cronjob.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if and .Values.cull.enabled .Values.cull.maxAge }}
```

Comment:
can we instead make this into a deployment that just maintains state itself? I think that's much easier for debugging purposes in cases like this (just look at logs) than a cronjob.

Reply:
Sure! That's what I had first, but I thought a cron job might make more sense than a long-running job always squatting resources. Inspecting the logs is definitely easier that way, though, so I'll switch it back.